### PR TITLE
Scaffold documentation for stomach and colon

### PR DESCRIPTION
**Pull request by mlin865**

### CODE REVIEW

File: docs/scaffolds/stomach.rst [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+
+Coordinates
+-----------
+
+The stomach scaffold defines the geometric, split and material coordinates.
+
+The geometric ``coordinates`` field gives an approximate, idealized representation of the stomach shape for the species,
+which is intended to be fitted to actual data for a specimen.
+
+The ``split coordinates`` follows the geometric field but provides continuous derivatives for nodes on the margin
```
Comment (rchristie) [inline docs/scaffolds/stomach.rst:56]: ... provides disconnected and duplicated derivatives for nodes on the margin ...

File: docs/scaffolds/stomach.rst [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+between the dorsal and ventral stomach (the boundary where stomach specimens are usually dissected into halves for
+imaging purposes). This field is intended for fitting data obtained from the dorsal or ventral stomach.
+
+The material coordinates field ``stomach coordinates`` defines a highly idealized coordinate system to give permanent
+locations for embedding structures in the stomach. It is defined by a pill-shaped structure with an inlet cylindrical
```
Comment (rchristie) [inline docs/scaffolds/stomach.rst:61]: 'capsule-shaped' is a better term; pill shape can be flat on one side.

File: docs/scaffolds/stomach.rst [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+tube representing the esophagus and another outlet tube representing the duodenum. This can be viewed by
+visualising this field in the *Display* tab of **Scaffold Creator** or by switching to the special ``Material``
+parameter set.
+
+The stomach scaffold supports limited refinement/resampling by checking *Refine* (set parameter to ``true``) with chosen
+*Refine number of elements* parameter. Be aware that only the ``coordinates`` field is currently defined on the refined
```
Comment (rchristie) [inline docs/scaffolds/stomach.rst:67]: *Refine number of elements* parameters (plural)

**Review by rchristie — CHANGES_REQUESTED** [review]
Great - some minor changes only,

**Comment by mlin865** [comment]
All changes made and ready for review again. Thanks!

**Review by rchristie — APPROVED** [review]
Great!

**Pull request merged**